docs/docs/adapters/circom.md

```diff
@@ -130,12 +130,12 @@ The Circom adapter exposes the following functions to be used in the iOS project
 ```swift
 // Generate a proof for a given circuit zkey, as well as the circuit inputs
 // Make sure that the name of the zkey file matches the one you set in the `set_circom_circuits!` macro
-generateCircomProof(zkeyPath: zkeyPath, circuitInputs: inputs) -> GenerateProofResult
+generateCircomProof(zkeyPath: zkeyPath, circuitInputs: input, proofLib: proofLib) -> GenerateProofResult
```

Comment:
It's minor, but I would suggest to let `proofLib` be the first argument.

Reply:
I even want to make it an optional input
like in typescript, you can set an input to be default value
```ts
generateCircomProof (
   zkeyPath: string, 
   circuitInputs: string, 
   proofLib: ProofLib = ProofLib.arkworks
)
```
so users can use
either 
`generateCircomProof(zkeyPath, inputs)`
or 
`generateCircomProof(zkeyPath, inputs, ProofLib.rapidsnark)`

not sure if rust can do something like this
so we don't have to change the previous API

Reply:
Unfortunately, Rust doesn't support default function agruments. 😢